app/uk/gov/hmrc/plasticpackagingtax/registration/forms/JobTitle.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms
+
+import play.api.data.Forms.{mapping, text}
+import play.api.data.{Form, Forms}
+import play.api.libs.json.Json
+
+case class JobTitle(value: Option[String]) {}
```

Comment:
Why are we making `JobTitle` optional? If `JobTitle` is required then this should be mandatory 
This should be sufficient.
```
case class JobTitle(value: String)
```
with Idiomatic Scala and _brevity_ we do not need the curly braces.

Reply:
done